Packs/Core/Playbooks/playbook-A_mail_forwarding_rule_was_configured_in_Google_Workspace.yml

```diff
@@ -0,0 +1,1784 @@
+          "x": 440,
+          "y": 2580
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: 252d8473-0602-420c-8fef-df880efcc695
+    type: regular
+    task:
+      id: 252d8473-0602-420c-8fef-df880efcc695
+      version: -1
+      name: Check if the rule was created outside of working hours
+      description: Checks whether the given value is within the specified time (hour) range.
+      scriptName: BetweenHours
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "45"
+    scriptarguments:
+      begin_time:
+        simple: "22:00:00"
+      end_time:
+        simple: "06:00:00"
+      extend-context:
+        simple: IsOutOfWorkingHours=
+      value:
+        complex:
+          root: Core.OriginalAlert.raw_abioc.event
+          accessor: event_timestamp
+          transformers:
+          - operator: TimeStampToDate
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 0,
+          "y": 765
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "22":
+    id: "22"
+    taskid: a1b2240d-c96c-46a2-8749-c94f8a214538
+    type: regular
+    task:
+      id: a1b2240d-c96c-46a2-8749-c94f8a214538
+      version: -1
+      name: Check if the user connected from an unusual geolocation
+      description: |-
+        Returns all elements from the left side that have a substring that is equal to an element from the right side. Note: This filter is case-insensitive.
+        E.g -AnyMatch left=baby right=A will return baby. For more examples see the filter's Readme.
+      scriptName: AnyMatch
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "43"
+    scriptarguments:
+      extend-context:
+        simple: IsAbnormalGeolocation=
+      left:
+        simple: ${Core.OriginalAlert.raw_abioc.event.saas_caller_ip_geolocation_days_seen_count},${Core.OriginalAlert.raw_abioc.event.service_caller_ip_asn_days_seen_count}
+      right:
+        simple: "0"
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 440,
+          "y": 765
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "25":
+    id: "25"
+    taskid: 3023b302-dc80-4e51-8b44-4489de9d410c
+    type: regular
+    task:
+      id: 3023b302-dc80-4e51-8b44-4489de9d410c
+      version: -1
+      name: Gmail - Remove forwarding email address
```

Comment:
Change to "Remove forwarding email address" for Gmail - this is unnecessary as this playbook handles Google workspace only.